public/app/plugins/panel/geomap/layers/data/StyleEditor.tsx

```diff
@@ -305,9 +305,9 @@ export const StyleEditor: FC<StandardEditorProps<StyleConfig, StyleEditorOptions
               value={value?.textConfig?.textBaseline ?? defaultStyleConfig.textConfig.textBaseline}
               onChange={onTextBaselineChange}
               options={[
-                { value: TextBaseline.Top, label: TextBaseline.Top },
-                { value: TextBaseline.Middle, label: TextBaseline.Middle },
-                { value: TextBaseline.Bottom, label: TextBaseline.Bottom },
+                { value: TextBaseline.Top, label: "Top" },
+                { value: TextBaseline.Middle, label: "Middle" },
+                { value: TextBaseline.Bottom, label: "Bottom" },
```

Comment:
We could use the [`capitalize` method](https://www.geeksforgeeks.org/lodash-_-capitalize-method/) from `lodash` library to avoid use of magic strings here
```suggestion
                { value: TextBaseline.Bottom, label: capitalize(TextBaseline.Bottom) },
```

Reply:
Currently we use hardcoded values everywhere else -- ideally the options should be part of the cue/gen process

But no strong feelings here